antora-ui-camel/src/css/frontpage.css

```
@@ -217,6 +218,12 @@ section.frontpage h1 {
   width: 27rem;
 }
 
+.packedFunctionalityIcon {
```

Needs to be scoped and use naming like in the rest of the CSS

```suggestion
.box .feature-icon {
```